bin/citgm-all.js

```diff
@@ -77,7 +77,7 @@ function runCitgm (mod, name, next) {
   process.on('SIGINT', cleanup);
   process.on('SIGHUP', cleanup);
   process.on('SIGBREAK', cleanup);
-  
```

Comment:
can you please remove these changes to white space. Please feel free to open a single PR to remove all erroneous white space in the project with a lint rule